klog.go

```diff
@@ -812,14 +812,17 @@ func (l *loggingT) output(s severity, log logr.InfoLogger, buf *buffer, file str
 		}
 	}
 	data := buf.Bytes()
-	if log != nil {
-		// TODO: set 'severity' and caller information as structured log info
-		// keysAndValues := []interface{}{"severity", severityName[s], "file", file, "line", line}
+	// In later version, we should check argument s is either errorLog or infoLog.
+	// However for now, as backward compatibility, all s other than errorLog goes to l.logr.Info().
+	// if l.logr != nil && (s == errorLog || s == infoLog) {
```

Comment:
this needs unit tests to verify that the backwards compatibility is preserved.

Reply:
Thanks for review.

OK. I'll create unit tests.